tests/test_models.py

```
@@ -2,25 +2,34 @@
 from datetime import datetime
 from pybanana.models.member import Member, Moderator
 from pybanana.models.common.buddy import Buddy, SubjectShaper
-from pybanana.models.common.managers import ManagerRecord
-from pybanana.models.common.moderators import ModeratorRecord
-from pybanana.models.common.credits import Credits, Author, CreditGroup, AffiliatedStudio
+from pybanana.models.common.managers import Manager
+from pybanana.models.common.moderators import Moderator
```

Ruff (0.12.2)

Redefinition of unused `Moderator` from line 3

(F811)